Sources/JWTKit/Vendor/FirebaseAuthIdentityToken.swift

```diff
@@ -0,0 +1,140 @@
+#if !canImport(Darwin)
+    import FoundationEssentials
+#else
+    import Foundation
+#endif
+
+public struct FirebaseAuthIdentityToken: JWTPayload {
+
```

Comment:
Nit: could you remove this whitespace along with the others to conform to the rest of the repo?